src/preregister.js

```diff
@@ -25,20 +25,25 @@ class PreRegister extends React.Component {
                         </div>
                         <div id="hackathon-description">
                             RIT's Premiere Hackathon
-                        </div>                        
+                        </div>
                         <div id="hackathon-info">
                             Feb 20-21 | ROCHESTER INSTITUTE OF TECHNOLOGY
                         </div>
-                        
+
                         <img className="hero-img desktop-hide" src={HeroImage} alt="A decorative background hero graphic."/>
-            
+
                         <MailchimpInput/>
 
-                        <CovidPopover 
+                        <div id="contact">
```

Comment:
I would suggest adding 'mobile-hide' to its classes so this doesn't mess with spacing on mobile devices
Right now it has no size-scaling so it looks big on smaller devices & pushes things down (the mobile COVID notice is definitely off-screen for iPhone 5/6/7) 
Alternatively I can play around with vertical spacing if you think it's important to have this be visible on mobile